connectors/pom.xml

```diff
@@ -104,6 +104,9 @@
                     <exampleSinkPropertiesFileTemplate>${project.basedir}/camel-kafka-connector-template-example-sink-properties.template</exampleSinkPropertiesFileTemplate>
                     <!-- WARNING: connectors not in catalog will be removed -->
                     <removeMissingComponents>true</removeMissingComponents>
+                    <exludedKamelets>
```

Comment:
I think the placeholder is wrong. It's excludedKamelets looking at the MOJO

Reply:
good catch tnx!